SofaKernel/framework/sofa/core/objectmodel/Base.h

```diff
@@ -332,17 +334,13 @@ class SOFA_CORE_API Base
 
     /// @}
 
-    /// @name logs
-    ///   Messages and warnings logging
-    /// @{
 
 private:
-
     /// effective ostringstream for logging
     mutable std::ostringstream _serr, _sout;
+    mutable std::deque<sofa::helper::logging::Message> m_messageslog ;
```

Comment:
So the component is still storing its own messages?
Is it only for this dev step?